docs/get-started/index.md

```diff
@@ -12,17 +12,6 @@ Athletes <athletes>
 
 ```
 
-```{toctree}
-:hidden:
```

Comment:
this just moves the contributing section from get started to it's own top level section. it already is in it's own folder so this is just a structural TOC change.